STATUS_GO_SHA256

```diff
@@ -1,3 +1,2 @@
 ## DO NOT EDIT THIS FILE BY HAND. USE `scripts/update-status-go.sh <tag>` instead
-
-1bhwwmypb64kf180kp634jwfppd3h6yda3qyh2icj3lz7y10pgvp
```

Comment:
Pretty sure that empty line is necessary. See `scripts/update-status-go.sh`.

Reply:
seems to be working fine without (android builds were built correctly, the rest is because I haven't removed some binding from status-go), but I will add it back just to make sure

Reply:
Fair enough.